test/e2e/network_segmentation.go

```diff
@@ -259,14 +259,15 @@ var _ = Describe("Network Segmentation", func() {
 							}, 5*time.Second).Should(BeTrue())
 						}
 
-						By("asserting healthcheck works (kubelet can access the UDN pod)")
-						// The pod should be ready
-						Expect(podutils.IsPodReady(udnPod)).To(BeTrue())
```

Comment:
was this the place you are seeing flakes at?

Reply:
Yup, added example failure to PR description.